duecredit/stub.py

```diff
@@ -57,7 +57,7 @@ def _donothing_func(*args, **kwargs):
         raise RuntimeError(
             "Imported due lacks .cite. DueCredit is now disabled")
 except Exception as e:
-    if type(e).__name__ != 'ImportError':
+    if type(e).__name__ not in ('ImportError', 'ModuleNotFoundError'):
```

Comment:
Why is `isinstance(e, ImportError)` not a better check, since [`ModuleNotFoundError`](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError) is a subclass of `ImportError`?

Reply:
Good point - thanks @effigies !  I found no counter-argument or some comment/commit description which would warrant name comparison here.  So I will change in duecredit first, and eventually we could update a stub here.